src/DistributedFFT.chpl

```diff
@@ -213,11 +141,17 @@ prototype module DistributedFFT {
         timeTrack.start();
         forall iy in offset(ySrc) {
           zPlan.execute(myplane[0, iy, zSrc.first]);
-          // This is the transpose step
-          copy(Dst[iy, ix, zSrc.first], myplane[0, iy, zSrc.first], myLineSize);
-          // If not last slice, copy over
-          if (ix != xSrc.last) {
-            copy(myplane[0, iy, zSrc.first], Src[ix+1, iy, zSrc.first], myLineSize);
+          // Transpose data into Dst, and copy the next Src slice into myplane
```

Comment:
A completely minor preference (which you can ignore) -- thinking about this, I would put the `if useElegant` right at the communication site and repeat it rather that repeating the `if (ix != xSrc.last)`. Seems a little more clear that the algorithm is identical.

Reply:
Will change -- I'm happy to defer to your preferences, especially since it's your code

Reply:
Yeah, but my preferences are sometimes ill-informed. :-)